plugin/pkg/admission/podpreset/admission.go

```diff
@@ -38,6 +38,7 @@ import (
 
 const (
 	annotationPrefix = "podpreset.admission.kubernetes.io"
+	exclusionSuffix  = "PodPresetOptOut"
```

Comment:
@derekwaynecarr where is the alpha text coming from? https://github.com/kubernetes/kubernetes/blob/master/plugin/pkg/admission/podpreset/admission.go#L40
Should the new annotation have alpha or not?